sympy/stats/frv.py

```diff
@@ -321,6 +340,13 @@ def compute_quantile(self, expr):
         return Lambda(p, Piecewise(*set))
 
     def probability(self, condition):
+        rvs = random_symbols(condition)
+        for rv in rvs:
+            if hasattr(rv.pspace.args[1], 'is_symbolic') and \
+                rv.pspace.args[1].is_symbolic:
```

Comment:
this code has been repeated three times, can you find a way to avoid code repetitions?